tests/integration_test.py

```diff
@@ -285,13 +279,29 @@ def test_ro(self):
             logs = logs.decode('utf-8')
         self.assertIn(self.filename, logs)
 
-        # FIXME: format changes in API version >= 1.20
+        self.assertIn(self.filename, logs)
```

Comment:
This seems to be a duplicate of the line above - merge conflict resolution mishap?